pkg/storage/kubernetes.go

```diff
@@ -80,10 +80,12 @@ func (k kubernetesClient) List() (StorageList, error) {
 	}
 
 	localStorage := ConvertListLocalToMachine(localConfigStorage)
-
-	clusterStorage, err := k.ListFromCluster()
-	if err != nil {
-		return StorageList{}, err
+	var clusterStorage StorageList
+	if k.client.GetKubeClient() != nil {
```

Comment:
same as https://github.com/openshift/odo/pull/4472/files#r587362266

Reply:
no actually we cannot because if you observe here https://github.com/openshift/odo/pull/4472/files#diff-491c294311c401a333e96f63b7fdf3a40f2f16a0d014bd7806d7539e7a99a85cR66, the receiver is not a pointer which means in any scenerio the whole client would never be `nil` as its not a pointer.